### DISCUSSION TITLE
Page speed score drastic drop on mobile

### DISCUSSION THREAD

**Discussion by afsanefda** [GoogleChrome/lighthouse#12732] [Q&A]
We have faced an issue on our mobile scores since middle of May. The score dropped drastically from average of 71 to 34. I checked the latest releases **V7.5 - released 18 May 2021** and **V8 - released 2 Jun 2021**. I was wondering if you have any idea what was the major root cause for this ? I mean there are plenty of changes but not pretty sure about the root cause. 
Thank you in advance!

**Comment by patrickhulce** [comment]
The [v8 performance changes FAQ document](https://github.com/GoogleChrome/lighthouse/blob/master/docs/v8-perf-faq.md#give-me-a-summary-of-the-perf-score-changes-in-v80-whats-newdifferent) answers this quite nicely :)
